jazzy/distribution.yaml

```
@@ -2572,6 +2572,24 @@ repositories:
       url: https://github.com/tier4/heaphook.git
       version: main
     status: maintained
+  hector_rviz_overlay:
+    source:
+      type: git
+      url: https://github.com/tu-darmstadt-ros-pkg/hector_rviz_overlay.git
+      version: jazzy
+    status: maintained
+  hector_rviz_overlay_controls:
+    source:
+      type: git
+      url: https://github.com/tu-darmstadt-ros-pkg/hector_rviz_overlay.git
+      version: jazzy
+    status: maintained
+  hector_rviz_overlay_demo:
+    source:
+      type: git
+      url: https://github.com/tu-darmstadt-ros-pkg/hector_rviz_overlay.git
+      version: jazzy
+    status: maintained
```

Since the three packages seem to be in the same repo I believe we don't need to add them individually but adding just one main source will do:

```suggestion
  hector_rviz_overlay:
    source:
      type: git
      url: https://github.com/tu-darmstadt-ros-pkg/hector_rviz_overlay.git
      version: jazzy
    status: maintained
```

---

Isn't that file used by rosdep to resolve the keys?

---

Never mind, some people use one entry per package, but SMACC, for example, only adds the repo and specifies the contained packages in the release.

---

I thought the later you mentioned is the generic way and adds less clutter to the rosdistro. Thanks for making the changes!